tests/PhpBrew/Command/EnvCommandTest.php

```diff
@@ -25,5 +25,11 @@ public function testEnvCommand()
         $this->assertCommandSuccess("phpbrew env");
     }
 
-
+    /**
+     * @outputBuffering enabled
+     */
+    public function testEnvCommandEmptyArg()
+    {
+        $this->assertCommandSuccess("phpbrew env ");
```

Comment:
The test doesn't reflect the issue since the trailing space is insignificant and the test won't use any shell to run the command.